distrobox-enter

```diff
@@ -531,5 +531,4 @@ fi
 
 # Generate the exec command and run it
 cmd="$(generate_command)"
```

Comment:
If we do a `cmd="$(generate_command | tr '\n' ' ')"` we can get rid of the `\\` in the code, and quoted eval will continue to work :+1:

Reply:
I think that the intended newlines are also unexpectedly converted to spaces in that way. For example, the newlines specified by the user in the command line would be converted to spaces, e.g., in the case `git commit -m $'do something\n\nhello world'`.

Reply:
hmmm you're right, just moving it to line 391:

```sh
	result_command="$(echo "${result_command}" | tr '\n' ' ')"
	if [ -n "${container_command}" ]; then
		result_command="${result_command} ${container_command}"
	else
		# if no command was specified, let's execute a command that will find
		# and run the default shell for the user
		result_command="${result_command} sh -c \"\\\$(getent passwd ${USER} | cut -f 7 -d :) -l"\"
	fi
```

This fixes the issue